packages/podge/podge.0.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: [ "Edgar Aroutiounian <edgar.factorial@gmail.com>" ]
+license: "GPL-3.0"
```

Comment:
The most permissive license that let's people use the library without any fuss is the BSD or ISC license (e.g. what MirageOS uses: e.g. https://github.com/mirage/ocaml-cohttp/blob/master/LICENSE ).

Every new version of the library requires a new pull request, but `opam-publish` is available to make this workflow minimum effort for you -- please do feel free to register bug reports if you have permissions problems with it and we'll do our best to fix them.